upload/upload.c

```diff
@@ -33,8 +37,8 @@ void configureSerialPort(FlashState* s, int baud) {
   if (s->serialPort != -1) {
     close(s->serialPort);
   }
-  s->serialPort = open_serial(SERIAL_PORT_PATH, fd_convertBaud(baud));
-}
+  s->serialPort = fd_openSerial(SERIAL_PORT_PATH, MTK7697_BAUD);
```

Comment:
It's better to use the baud rate passed into the function and not the define.

Reply:
Gotcha and fixed